rafka.go

```diff
@@ -40,6 +40,8 @@ var (
 func main() {
 	app := cli.NewApp()
 	app.Name = "rafka"
+	app.Usage = "Kafka with a Redis API"
+	app.Description = "A gateway service that exposes Kafka using the Redis protocol"
```

Comment:
Unfortunately we're pinned to urfave/cli@0.18.1, while the `Description` field was added in 1.19. So we can't use this yet.

I've pushed https://github.com/skroutz/rafka/commit/9516682f3b8b35d6df83b8eba584049b4da40ca3 to make this clear.

The `Usage` is fine though.